packages/office-addin-usage-data/test/test.ts

```diff
@@ -162,13 +163,20 @@ describe("Test office-addin-usage data-package", function() {
       addInUsageData.setUsageDataOff();
```

Comment:
This is a poorly written unit test. We should have a test case group for the maskFilePaths() function, and separate test cases. It should follow a pattern of arrange, act, assert. It would be great to see improvements made here.

Reply:
Sure, I will create a new test group for this function